### DISCUSSION TITLE
Disable "All (x)" text when all options are selected

### DISCUSSION THREAD

**Discussion by xella29** [sa-si-dev/virtual-select#51] [Q&A] [answered]
Hello ! 

I would like to know if it's possible to :
- add a propertie option to disable the tool to write "All (x)" in the "vscomp-value" value when we select all options (and only write the label of selected values, or "x + "optionsSelectedText" value".
and also to :
- add a propertie option to choose the string to show when we select all options (for exemple to remplace "All (x) with "All countries (x)").

If these two options are not developped yet, it could be great to add these features :) 

Thanks for your answer !

**Comment by sa-si-dev** [comment]
@xella29 Thank you for the suggestions.

I will implement them in the future release.

**Reply by xella29** [reply]
No problem ! It would be so nice if you can leave me a message here when these features will be added, so i could add it to my project :) !

Thank you very much !

**Comment by sa-si-dev** [comment, marked as answer]
@xella29, Features you have requested are available in the [release v1.0.14](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.14)

1) disableAllOptionsSelectedText
2) allOptionsSelectedText

**Comment by xella29** [comment]
Thank you so much for your reactivity !!! :) 

It's working perfectly :) ! Great project, thank you so much for it :)
